src/LinearSolverTypes.f90

```diff
@@ -581,8 +594,13 @@ SUBROUTINE init_LinearSolverType_Base(solver,Params,A)
               CALL PCSetType(solver%pc,PCJACOBI,iperr)
             ELSEIF(TRIM(PreCondType)=='BJACOBI_ILU') THEN
               CALL PCSetType(solver%pc,PCBJACOBI,iperr)
+#if ((PETSC_VERSION_MAJOR>=3) && (PETSC_VERSION_MINOR>6))
+              CALL PetscOptionsSetValue(PETSC_NULL_OPTIONS,"-sub_ksp_type","preonly",iperr)
+              CALL PetscOptionsSetValue(PETSC_NULL_OPTIONS,"-sub_pc_type","ilu",iperr)
```

Comment:
What's the extra first argument doing?

Reply:
![image](https://user-images.githubusercontent.com/15932146/80138514-e8fecc80-8572-11ea-9a53-c4c286f8fc76.png)

Reply:
Gotcha.  Weird.